src/features/channels/services/channel.service.ts

```diff
@@ -4,14 +4,17 @@ import { Channel } from '../types/channel';
 // TODO: Define the correct url, dummy for demonstration
 const findAll = () => {};
 
-const findByUser = async () => {
-  const response = await httpService.get('/pokemon?offset=0&limit=5');
-  return response.data.results as Channel[];
+const findChannels = async () => {
+  const config = {
+    headers: { Authorization: `Bearer ${process.env.TOKEN}` },
```

Comment:
Please add a TODO comment here to say we are not supposed to add tokens in every request, and we are not supposed to use the token from the env variable. Add a new task to our board to not lose track of it.